### PR TITLE
Added "az registry" commands for RP operations

### PR DESCRIPTION
**Pull request by djyou**
Commands include create/delete/show/list/update container registries. Added SDK for Python code for container registries.

### CODE REVIEW

File: src/azure/cli/_azure_env.py [hunk 1]
```diff
@@ -5,9 +5,7 @@
 
 CLIENT_ID = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
 
-ENV_DEFAULT = 'AzureCloud'
-ENV_US_GOVERNMENT = 'AzureUSGovernment'
-ENV_CHINA = 'AzureChinaCloud'
+ENV_DEFAULT = 'dogfood'
```
Comment (sivagms) [inline src/azure/cli/_azure_env.py]: Curious, why didn't you just add one more env instead of replacing the existing ones..

**Comment by sivagms** [comment]
:ship:

**Comment by sivagms** [comment]
:shipit:

File: src/azure/cli/_azure_env.py (hunk 1 above)
Comment (djyou) [inline src/azure/cli/_azure_env.py]: The CLI does not support adding/switching environments yet and ENV_DEFAULT is used in multiple places directly. We will need to update all the places that use ENV_DEFAULT if we add one more. This is basically a temporary solution while involving as few lines of code as possible.

**Comment by DavidObando** [comment]
Looks good to me.

**Pull request merged**